### PR TITLE
fix stantard/standard typo

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: aioconsole/stream.py [hunk 1]
```diff
@@ -159,9 +159,12 @@ async def open_stantard_pipe_connection(pipe_in, pipe_out, pipe_err, *, loop=Non
     return in_reader, out_writer, err_writer
 
 
+open_stantard_pipe_connection = open_standard_pipe_connection
```
Comment (vxgmichel) [inline aioconsole/stream.py:162]: There's no need for backward compatibility here

Comment (graingert) [reply aioconsole/stream.py:162]: Can you use a suggestion? Then I can apply it as I'm afk

File: aioconsole/stream.py [hunk 2]
```diff
@@ -159,9 +159,12 @@ async def open_stantard_pipe_connection(pipe_in, pipe_out, pipe_err, *, loop=Non
...
+
+
```
Comment (vxgmichel) [inline aioconsole/stream.py:164]: Here you go!

```suggestion
```

Comment (graingert) [inline aioconsole/stream.py:164]:
```suggestion
```

**Comment by vxgmichel** [comment]
Merged! Thanks @graingert :)

**Pull request merged**